apps/cookbook/src/app/examples/grid-layout-example/grid-layout-core-example/grid-layout-core-example.component.scss

```diff
@@ -1,10 +1,8 @@
+@import './libs/designsystem/src/lib/scss/utils';
+
 // Grid
 $columns: 12;
```

Comment:
We could consider if we should have specific design tokens for grid in `_variables.scss`. 

But perhaps this should be talked more over and a new issue should be created for this